gravatar/README.md

```diff
@@ -30,12 +30,14 @@ Gravatar lets users self-rate their images to be used at appropriate audiences.
 See more information at [Gravatar][1].
 
 ## Alternative Configuration
-Open the `config/local.config.php` file and add "gravatar" to the list of activated addons:
+Open the `config/node.config.php` file and add "gravatar" to the list of activated addons:
 
-	'system' => [
-		...
-		'addon' => '...,gravatar'
+	'addons' => [
 		...
+		'gravatar' => [
+			last_update => 1672865741,
```

Comment:
Isn’t it easier to put `0` and let the addon update routine take care of the field? Or omit the field entirely here?